### PR TITLE
grow-521 removing the loginButtonExperimentShown and setting button t…

### PR DESCRIPTION
**Pull request by BoulderBrains**
…ext to Continue

These changes cover the requested R1 changes for [grow-521](https://kiva.atlassian.net/browse/GROW-521)

The request was to change the button text from "login to continue" to "continue" to be more inclusive of the guest checkout scenario. 

- The button text was being controlled by the loginButtonExperimentShown setting. I spoke to Brandon who said that experiment was no longer being used, so I removed it and hardcoded the "continue" text for the button.

-  I think I cleaned up all the loose ends removing the exp, let me know if you see something I missed. 

🙏 Thanks for the review.

### CODE REVIEW

File: src/pages/Checkout/CheckoutPage.vue [hunk 1]
```diff
@@ -433,24 +433,9 @@ export default {
 			return false;
 		},
 		checkoutSteps() {
-			if (this.loginButtonExperimentShown) {
-				return [
-					'Basket',
-					'Payment',
-					'Thank You!'
-				];
```
Comment (emuvente) [inline src/pages/Checkout/CheckoutPage.vue:441]: I think we want this version of the checkout steps.

File: src/pages/Checkout/CheckoutPage.vue [hunk 2]
```diff
@@ -433,24 +433,9 @@ export default {
...
-				'Basket',
-				'Account',
-				'Payment',
-				'Thank You!'
-			];
+			return ['Basket', 'Account', 'Payment', 'Thank You!'];
 		},
 		currentStep() {
-			if (this.loginButtonExperimentShown) {
-				return this.isLoggedIn ? 1 : 0;
```
Comment (emuvente) [inline src/pages/Checkout/CheckoutPage.vue:452]: We also want this to decide the current step

**Comment by BoulderBrains** [comment]
Thanks @emuvente I kept the wrong conditions there. All fixed up.

**Review by emuvente — APPROVED** [review]
👍

**Review by ryan-ludwig — APPROVED** [review]

**Pull request merged**